ingest/cell_metadata.py

```diff
@@ -150,28 +158,17 @@ def validate_type_annotations(self):
         :return: boolean   True if valid, False otherwise
```

Comment:
Can you add a little more information about what this function is doing?

Reply:
added better docstring content in `29aa7e3`